pkgs/development/python-modules/ush/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi }:
+buildPythonPackage rec {
+  pname = "ush";
+  version = "3.1.0";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    sha256 = "JB56zVbn4+xeSNG/D3At4qgHYjMM661E1FfPbeXRtbQ=";
+  };
+
+  pythonImportsCheck = [ "ush" ];
```

Comment:
Do we have tests?

Reply:
[Yes](https://github.com/tarruda/python-ush/tree/master/tests), and they seem mostly tame but I started getting lazy this deep in the dependency tree after `pytestCheckHook` didn't work for some reason.

Reply:
Okay, bashing it a bit more fixed that. (: